backend/xivdpscalc/character/__init__.py

```diff
@@ -43,6 +44,13 @@ def get_gcd(self) -> float:
         """
         return math.floor(0.25 * (1000 - self.stats[Stats.SPEED].get_multiplier())) / 100
 
+    def get_cast_time(self, base_cast_time: CastTime) -> CastTime:
+        """
+        Returns the character's gcd given its skill or spell speed
+        :returns: the gcd in seconds
+        """
+        return math.floor(base_cast_time / 10 * (1000 - self.stats[Stats.SPEED].get_multiplier())) / 100
```

Comment:
you can skip the math.floor by using `//`:

this should be equivalent: `base_cast_time * (1000 - self.stats[Stats.SPEED].get_multiplier()) // 10 / 100`